AWSKinesis/AWSAbstractKinesisRecorder.m

```diff
@@ -273,12 +273,19 @@ - (AWSTask *)submitAllRecords {
 
                     NSString *streamName = temporaryRecords[0][@"stream_name"];
 
-                    [[self.recorderHelper submitRecordsForStream:streamName
-                                                         records:temporaryRecords
-                                                   partitionKeys:partitionKeys
-                                                putPartitionKeys:putPartitionKeys
-                                              retryPartitionKeys:retryPartitionKeys
-                                                            stop:&stop] waitUntilFinished];
```

Comment:
We had an invalid pool identity set on the configuration, and the error was being swallowed here.